jenkins/run-server-func-tests

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+export EXTRA_PODMAN_SWITCHES=${EXTRA_PODMAN_SWITCHES:-"--pull=always -e=COV_REPORT_XML=yes"}
+export IMAGE_KIND=${IMAGE_KIND:-"fedora"}
+export IMAGE_REPO=${IMAGE_REPO:-"quay.io/pbench"}
+export IMAGE_ROLE=${IMAGE_ROLE:-"ci"}
+
+export PB_CONTAINER_REG=${PB_CONTAINER_REG:-"images.paas.redhat.com"}
+export PB_REPO_NAME=${PB_REPO_NAME:-"pbench"}
+
+export PB_ELASTICSEARCH_IMAGE_NAME=${PB_ELASTICSEARCH_IMAGE_NAME:-"elasticsearch"}
+export PB_ELASTICSEARCH_IMAGE_TAG=${PB_ELASTICSEARCH_IMAGE_TAG:-"7.13.3"}
+export PB_POSTGRESQL_IMAGE_NAME=${PB_POSTGRESQL_IMAGE_NAME:-"postgresql-13"}
+export PB_POSTGRESQL_IMAGE_TAG=${PB_POSTGRESQL_IMAGE_TAG:-"latest"}
+export PB_SERVER_IMAGE_NAME=${PB_SERVER_IMAGE_NAME:-"pbench-server-ci"}
+export PB_SERVER_IMAGE_TAG=${PB_SERVER_IMAGE_TAG:-"main"}
```

Comment:
You like `"main"` as a default better than the contents of the `jenkins/branch.name` file?

Reply:
Good point: we almost always want the container we just built in the CI pipeline, so I'm not sure there's much value in a different default.

Reply:
The container we just built in the CI pipeline will be indicated by a pre-existing definition of `PB_SERVER_IMAGE_TAG`, so this question is moot in that instance.  (I.e., the CI only looks at `branch.name` for branch builds -- for PR's it uses the PR number and ignores the file -- and either way it passes the desired value via the environment variable.)  So, this only matters for someone running outside the CI...and, I imagine that they will always want `main` except when they don't.  😉

`"main"`  isn't a bad default...it's just that, in the past, we've always referenced `jenkins/branch.name` for this sort of thing.

Reply:
> You like `"main"` as a default better than the contents of the `jenkins/branch.name` file?

No, just transferred a default from one place (one of the `pod*.yml` files) to another.

I took your suggestion and set it to the `jenkins/branch.name`.

Note that the CI does not use the `jenkins/branch.name` file.

Reply:
> the CI does not use the `jenkins/branch.name` file.

Ah, right:  the CI gets the branch name (indirectly) from Git (via Jenkins), rather than from the file.  We could change that...if we wanted...(although, I think I prefer getting it from Git 🤷).

Reply:
See PR #3112.